composables/types/Page.ts

```diff
@@ -55,6 +55,12 @@ export interface ArticlePage extends Page {
 
 }
 
+export interface StaffPage extends Page {
```

Comment:
I was mimicking what was done for the Tags page... but I am not using this.. and not sure if I do need it.